### ISSUE TITLE
mac fonts don't read

### ISSUE THREAD

**Issue by luikluikluik** [fontforge/fontforge#4916]
fontforge doesn't read any of [these](https://github.com/PoomSmart/EmojiLibrary/releases/tag/0.14.5) fonts

**Comment by Bry10022** [comment]
I think that might be because of the sbix table, which FontForge probably does not support properly. The sbix table is comprised of bitmap images embedded into the table in the font file

**Comment by Bry10022** [comment]
> > I think that might be because of the sbix table, which FontForge probably does not support properly. The sbix table is comprised of bitmap images embedded into the table in the font file
> 
> is there some other tool that makes this conversion properly?
Not that I can think of. I wonder if Glyphs (Mac) can be able to read the fonts correctly? I can't test that cause I don't have a Mac…

**Comment by ctrlcctrlv** [comment]
You're correct, it's a matter of `sbix`. The general color fonts issue is #677, but this one can remain for `sbix`, unless there is an earlier issue I'm not seeing.
